.circleci/config.yml

```diff
@@ -3,79 +3,77 @@
 # Check https://circleci.com/docs/2.0/language-python/ for more details
 #
 
-version: 2
+version: 2.1
 workflows:
-  version: 2
+  version: 2.1
   test:
     jobs:
-      - test-3.6
-      - test-2.7
+      - test:
+          filters:
+            tags:
+              only: /v?[0-9]+(\.[0-9]+)*/
+          matrix:
+            parameters:
+              python_version: ["2.7.16", "3.5.7"]
```

Comment:
Since we don't have problems here with the node version, I think that we can use "2.7" and "3.5" in order to use the latest version of them.

If we use them, we wouldn't need the _manipulation_ to the TOXENV that we had to do in eox-core.

Reply:
@morenol  done